dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/IEC63406/Controls/BaseControls/CurrentLimitation.mo

```diff
@@ -0,0 +1,217 @@
+  Modelica.Blocks.Interfaces.RealInput qMaxPu(start = QMax0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-140, 40}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-134, 40}, extent = {{-14, -14}, {14, 14}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput qMinPu(start = QMin0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-140, -20}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-134, -40}, extent = {{-14, -14}, {14, 14}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput iQcmdPu(start = Q0Pu * SystemBase.SnRef / (SNom * U0Pu)) annotation(
+    Placement(visible = true, transformation(origin = {-140, -80}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-140, -100}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+
+  //Output variables
+  Modelica.Blocks.Interfaces.RealOutput iQMaxPu(start = IQMax0Pu) "Maximum reactive current" annotation(
+    Placement(visible = true, transformation(origin = {130, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {130, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealOutput iQMinPu(start = IQMin0Pu) "Minimum reactive current" annotation(
+    Placement(visible = true, transformation(origin = {130, 20}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {130, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealOutput iPMaxPu(start = IPMax0Pu) annotation(
+    Placement(visible = true, transformation(origin = {130, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {130, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  //Initial parameters
+  parameter Types.VoltageModulePu U0Pu "Initial voltage amplitude at grid terminal in pu (base UNom)" annotation(
+    Dialog(group = "Operating point"));
+  parameter Types.ActivePowerPu P0Pu "Initial active power at grid terminal in pu (base SnRef) (receptor convention)" annotation(
+    Dialog(tab = "Operating point"));
+  parameter Types.ReactivePowerPu Q0Pu "Initial reactive power at grid terminal in pu (base SnRef) (receptor convention)" annotation(
+    Dialog(tab = "Operating point"));
+  parameter Types.PerUnit IQMax0Pu "Initial maximum reactive current" annotation(
+    Dialog(tab = "Operating point"));
+  parameter Types.PerUnit IQMin0Pu "Initial minimum reactive current" annotation(
+    Dialog(tab = "Operating point"));
+  parameter Types.PerUnit IPMin0Pu annotation(
+      Dialog(tab = "Operating point"));
+  parameter Types.PerUnit IPMax0Pu annotation(
+      Dialog(tab = "Operating point"));
+  parameter Types.PerUnit QMax0Pu "Initial maximum reactive power" annotation(
+    Dialog(tab = "Operating point"));
+  parameter Types.PerUnit QMin0Pu "Initial minimum reactive power" annotation(
```

Comment:
Also, could you put initial parameters, parameters, input variables, output variables in alphabetical order?